cookiecutter/generate.py

```diff
@@ -41,7 +42,16 @@ def generate_context(context_file='cookiecutter.json', default_context=None,
     context = {}
 
     file_handle = open(context_file)
-    obj = json.load(file_handle, object_pairs_hook=OrderedDict)
+    try:
+        obj = json.load(file_handle, object_pairs_hook=OrderedDict)
+    except ValueError as e:
+        # JSON decoding error.  Let's throw a new exception that is more friendly for the developer
+        # or user.
+        full_fpath = os.path.abspath(context_file)
+        json_exc_message = str(e)
```

Comment:
I don't think so.  We aren't trying to debug the application here, we are trying to debug the json file.  A traceback would run counter to that purpose.